pkg/controller/operands/ssp.go

```diff
@@ -3,289 +3,125 @@ package operands
 import (
```

Comment:
Github's diff view of this file is terrible.
In essence, this file has been rewritten from scratch, so it better be viewed as a whole file, and compared to the old file as a whole.